app/gui/src/project-view/util/react.ts

```diff
@@ -0,0 +1,13 @@
+import { applyPureReactInVue } from 'veaury'
+import { type DefineComponent } from 'vue'
```

Comment:
`import type {}` vs `import { type }` please - the former is stripped completely, the latter leaves an import depending on bundler (some bundlers assume side-effect-ful imports)